doc/SONiC_QoS_Mgmt_HLD-1.1.md

```diff
@@ -0,0 +1,649 @@
+
+
+- DSCP to TC map  
+  No Open Config YANG available.  
+  Augment Open Config YANG model with new definitions similar to [SONIC YANG for DSCP-to-TC-map](https://github.com/project-arlo/sonic-mgmt-framework/blob/transformer-phase1/src/cvl/testdata/schema/sonic-dscp-tc-map.yang)
+
+
+- Dot1p to TC map  
+
+  No Open Config YANG available.  
+  Augment Open Config YANG model with new definitions similar to [SONIC YANG for DOT1P-to-TC-map](https://github.com/project-arlo/sonic-mgmt-framework/blob/transformer-phase1/src/cvl/testdata/schema/sonic-dot1p-tc-map.yang)
+
+
+- TC to Queue map  
+
+  No Open Config YANG available.   
+  Augment Open Config YANG model with new definition similar to [SONIC YANG for TC-to-Queue-map](https://github.com/project-arlo/sonic-mgmt-framework/blob/transformer-phase1/src/cvl/testdata/schema/sonic-tc-queue-map.yang)
+
+
+
+- TC to Priority group  
+
+  No Open Config YANG available.   
+  Augment Open Config YANG model with new definition similar to [SONIC YANG for TC-to-priority-group-map](https://github.com/project-arlo/sonic-mgmt-framework/blob/transformer-phase1/src/cvl/testdata/schema/sonic-tc-priority-group-map.yang)
+
+
+- PFC priority to Queue  
+
+  No Open Config YANG available.   
+  Augment Open Config YANG model with new definition similar to [SONIC YANG for PFC-pirority-to-queue-map](https://github.com/project-arlo/sonic-mgmt-framework/blob/transformer-phase1/src/cvl/testdata/schema/sonic-pfc-priority-queue.yang)
+
+
+- PFC Configuration
+
+  No open config support for enable PFC priorities on interface.
+  Augment Open Config YANG model with new attribute for PFC priorities and PFC Asynchronous mode.
+
+
+- Interface QoS  
+
+  openconfig-qos-interfaces.yang:: qos-interfaces-config
+  openconfig-qos-elements.yang:: qos-queue-config
+
+
+- QoS Statistics  
+
+  openconfig-qos-interfaces.yang:: qos-interfaces/interface/interface-id/output/queues/queue/state  
+  Augment Open Config YANG model with new definitions for queue statistics and watermark statistics.
+
+
+- QoS Clear Statistics  
+
+  No Open Config YANG available.   
+  RPC based sonic YANG model with new definitions for clear QOS statistics and watermark statistics.
+
+
+- Threshold Breach  
+
+  Augment Open Config YANG model with new definitions of Thresholds breach get.
+
+
+
+### 3.2.2 CLI
+
+#### 3.2.2.1 Configuration Commands
+
+- Configuring WRED
+````
+  sonic(config)# wred <name>
+  sonic(conf-wred-<name>)# random-detect color [green|yellow|red] minimum-threshold 100 maximum-threshold 300
+  drop-probability 40
+  sonic(conf-wred-<name>)# random-detect ecn <ecn_none/green/all>
+
+  sonic(config)# no wred <name>
+````
+- Configure Scheduler Policy for queues
+````
+  sonic(config)# scheduler-policy <name>
+  sonic(conf-sched-policy)# scheduler queue <q#>
+  sonic(conf-sched-policy-q)# priority
+  sonic(conf-sched-policy-q)# min <val>
+  sonic(conf-sched-policy-q)# min-burst <val>
+  sonic(conf-sched-policy-q)# max <val>

```

Comment:
```
sonic(conf-scheduler-test-queue-0)# min
  <0..400000000>  0 - 400000000 Kilo bits per second
```
Please note that CONFIG_DB expects cir/cbs/pir/pbs in Bytes per Second. I hope you are taking care of kbps to Bps conversion before writing to CONFIG_DB

Reply:
We are doing